src/SCRIPTS/BF/X7/x7pre.lua

```diff
@@ -8,9 +9,6 @@ PageFiles =
     "filters1.lua",
     "filters2.lua",
     "pwm.lua",
-    "vtx.lua",
-    "rescue.lua",
```

Comment:
These should not be removed.

Reply:
OMG did I do this?
I only wanted to change these on my own github 0crap account and not at all push into the official master. Only do the minor textual improvement back into the master. Can you tell me what I did wrong?